src/interfacekernels/InterfaceSorptionSievert.C

```diff
@@ -0,0 +1,160 @@
+/************************************************************/
+/*                DO NOT MODIFY THIS HEADER                 */
+/*   TMAP8: Tritium Migration Analysis Program, Version 8   */
+/*                                                          */
+/*   Copyright 2021 - 2023 Battelle Energy Alliance, LLC    */
+/*                   ALL RIGHTS RESERVED                    */
+/************************************************************/
+
+#include "InterfaceSorptionSievert.h"
+
+registerMooseObject("TMAPApp", InterfaceSorptionSievert);
+registerMooseObject("TMAPApp", ADInterfaceSorptionSievert);
+
+template <bool is_ad>
+InputParameters
+InterfaceSorptionSievertTempl<is_ad>::validParams()
+{
+  InputParameters params = InterfaceKernelParent<is_ad>::validParams();
+  params.addClassDescription("Computes Sievert's law at interface between solid and gas.");
```

Comment:
```suggestion
  params.addClassDescription("Computes Sievert's law at interface between solid and gas in isothermal conditions.");
```

Since I'm suggesting we use this in the doco in another comment, I would like to add this detail to the main class description.